Appraisals

```diff
@@ -28,3 +28,19 @@ appraise 'activerecord-5' do
   gem 'rspec-mocks'       , github: 'rspec/rspec-mocks'
   gem 'rspec-support'     , github: 'rspec/rspec-support'
 end
+
+appraise 'activerecord-6' do
+  gem "sqlite3", "~> 1.3.6", :platform => "ruby"
+  gem "activerecord", ">= 6.0.0.beta3", :require => "active_record"
```

Comment:
You could use `'>= 6.0.0.rc1'` now.

Reply:
I've just pushed another commit that does that. Looking forward to final :)